apps/app/src/containers/PlanCard.tsx

```diff
@@ -296,6 +296,19 @@ const PlanStatusDescription = ({
         );
       }
 
+      if (hasGithubPurchase) {
+        return (
+          <Paragraph>
+            You subscribed from GitHub Marketplace. You can upgrade or cancel
+            your plan directly from{" "}
+            <Anchor href="https://github.com/marketplace/argos-ci">
+              GitHub
+            </Anchor>
+            .
```

Comment:
Improved version:

Your subscription was made through the GitHub Marketplace. To manage your plan, please visit the <Anchor href="https://github.com/marketplace/argos-ci">Argos page on GitHub</Anchor>.

Reply:
I'm not sure that it's necessary because there is a link "Manage subscription" just bellow.

Reply:
No "Manage subscription" link is actually displayed (see Discord).

I think your version is just longer with the exact same meaning. I prefer to be shorter.

Reply:
> No "Manage subscription" link is actually displayed (see Discord).

Look like we have a bug here.

Reply:
I let you fix it, anyway showing "next payment" when it is from GitHub is not correct, it is GitHub business not ours.